roles/postgresql_client_access/README.md

```diff
@@ -86,4 +86,4 @@ Docker needs to be installed and configured.
 GNU Affero General Public License v3
 
 ## Author Information
-Famedly GmbH, famedly.de
\ No newline at end of file
+Famedly GmbH, famedly.de
```

Comment:
I'm a bit blind, where's  the diff here?

Reply:
Doing a local diff it shows that there was no newline at the end of the file before, no idea why that doesn't show in the github diff

Reply:
GitLab nicely showed that with a little indicator, and I thought github does the same, but apparently not.